io.github.alfianlosari.GTKChatGPT.json

```diff
@@ -0,0 +1,58 @@
+{
+    "app-id": "io.github.alfianlosari.GTKChatGPT",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "47",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.swift6"
+    ],
+    "command": "GTKChatGPT",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/swift6/bin",
+        "prepend-ld-library-path": "/usr/lib/sdk/swift6/lib"
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "GTKChatGPT",
+            "builddir": true,
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/alfianlosari/GTKChatGPT",
+                    "tag": "0.1.0",
+                    "commit": "5b7808ed0921feb447a6fd3c84e6dfae4dcf6e58"
+                },
+                "generated-sources.json"
+            ],
+            "build-commands": [
+                "./setup-offline.sh",
+                "swift build -c release --static-swift-stdlib --skip-update",
+                "install -Dm755 .build/release/GTKChatGPT /app/bin/GTKChatGPT",
+                "install -d /app/bin/GPTEncoder_GPTEncoder.resources",
+                "cp -r .build/release/GPTEncoder_GPTEncoder.resources/* /app/bin/GPTEncoder_GPTEncoder.resources/",
+                "install -Dm644 data/io.github.alfianlosari.GTKChatGPT.metainfo.xml /app/share/metainfo/io.github.alfianlosari.GTKChatGPT.metainfo.xml",
+                "install -Dm644 data/io.github.alfianlosari.GTKChatGPT.desktop /app/share/applications/io.github.alfianlosari.GTKChatGPT.desktop",
+                "install -Dm644 data/icons/io.github.alfianlosari.GTKChatGPT.png /app/share/icons/hicolor/scalable/apps/io.github.alfianlosari.GTKChatGPT.png"
```

Comment:
pngs go to 128x128, 256x256 etc. subfolders, scalable is for scalable icons.

Reply:
Updated